string name
std_msgs/ColorRGBA data